### PR TITLE
Section 04 if and loops

### PR DESCRIPTION
**Pull request by AndrePosm**
@VasylPosmit Can you please check and say , OK to do PR? Thx

### CODE REVIEW

File: course-sections/section-04-if/assets/scripts/app.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+function calculatedResult (calculationType) {
+  if (
+    calculationType !== "ADD" &&
+    calculationType !== "SUBTRACT" &&
+    calculationType !== "MULTIPLY" &&
+    calculationType !== "DIVIDE"
+    !enteredNumber
+  ) {
+    return;
```
Comment (VasylPosmit) [inline course-sections/section-04-if/assets/scripts/app.js:44]: Nice. It called "defensive programming". You are checking for correct argument.

However, you must not only "return". You must also throw an ERROR (like `console.error`)

File: course-sections/section-04-if/assets/scripts/app.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+function divide () {
+   calculatedResult ('DIVIDE');
+}
+
+
+addBtn.addEventListener('click', add);
+subtractBtn.addEventListener("click", subtract);
+multiplyBtn.addEventListener("click", multiply);
+divideBtn.addEventListener("click", divide);
```
Comment (VasylPosmit) [inline course-sections/section-04-if/assets/scripts/app.js:98]: Nice: this code is decent exercise 👍🏼

File: course-sections/section-04-if/assets/styles/app.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+#calculator button:active {
+  background: #084f88;
+  border-color: #084f88;
+}
+
+#calc-actions button {
+  width: 4rem;
+}
+
+#calc-actions {
```
Comment (VasylPosmit) [inline course-sections/section-04-if/assets/styles/app.css:76]: You need to learn to work with `css .classes` (Treehouse)

`id `is rarely used as overkill.

JFYI

File: project-2-part4/assets/scripts/app.js [hunk 4]
```diff
@@ -0,0 +1,258 @@
+const ATTACK_VALUE = 10;
```
Comment (VasylPosmit) [inline project-2-part4/assets/scripts/app.js:1]: This code should not be a part of Pull Request.

You must create a new branch from `main` . (During development, all feature-branches are created from `development` branch)

File: course-sections/section-04-if/assets/styles/app.css (hunk 3 above)
Comment (AndrePosm) [reply course-sections/section-04-if/assets/styles/app.css:76]: ok, ill do.

File: project-2-part4/assets/scripts/app.js (hunk 4 above)
Comment (AndrePosm) [reply project-2-part4/assets/scripts/app.js:1]: Yes , it was actually another project in session 2.... You're right.

**Pull request merged**